bigchaindb/__init__.py

```diff
@@ -30,7 +30,6 @@
     'port': int(os.environ.get('BIGCHAINDB_DATABASE_PORT', 27017)),
     'name': os.environ.get('BIGCHAINDB_DATABASE_NAME', 'bigchain'),
     'replicaset': os.environ.get('BIGCHAINDB_DATABASE_REPLICASET', 'bigchain-rs'),
-    'ssl': bool(os.environ.get('BIGCHAINDB_DATABASE_SSL', False)),
```

Comment:
Moved the MongoDB 'ssl' param from `_base_database_mongodb` to `_database_mongodb` as it is not base config for us in my opinion. The `login` and the `password` params can remain to authenticate as another user.
(My hunch is we might use it in the a soon to be subsequent PR).